tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -57,6 +58,14 @@ func RunTestPermutations(s *suite.Suite, testNamePrefix string, client *rancher.
 				s.Run(name, func() {
 					switch clusterType {
 					case RKE2ProvisionCluster, K3SProvisionCluster:
+						if testClusterConfig.CloudProvider == "aws" {
```

Comment:
We already have this aws string in the `provisioninginput` package with a stringer:

```suggestion
						if testClusterConfig.CloudProvider == provisioninginput.AWSProviderName.String() {
```